db_plugins/db/sql/connection.py

```diff
@@ -75,17 +96,34 @@ def connect(
             self.Session = self.Session
         else:
             self.Session = sessionmaker(bind=self.engine, **session_options)
```

Comment:
Incluyendo la línea 95: Cambiar el if/else por sólo un `if self.Session is None` que lleve a la creación. La asignación a sí mismo no es necesaria y puede ser confusa